esp32_s3_eye/idf_component.yml

```diff
@@ -1,20 +1,24 @@
-version: "2.1.3"
+version: "3.0.0"
 description: Board Support Package for ESP32-S3-EYE
 url: https://github.com/espressif/esp-bsp/tree/master/esp32_s3_eye
 
 targets:
   - esp32s3
 
 dependencies:
-  idf: ">=5.0"
+  idf: ">=4.4.5"
```

Comment:
Do we need some bugfix from 4.4.5 here? If not, I suggest setting the minimum required IDF version here.

Reply:
All BSPs, which is using `esp_lvgl_port` needs minimal version IDF 4.4.5. Because there is used API for registration callback.